### DISCUSSION TITLE
Configuration as Code for the following option: "Force all tables and views to be created in this schema when clicking CTAS or CVAS in SQL Lab."

### DISCUSSION THREAD

**Discussion by adberger** [apache/superset#20765] [Q&A / Help]
Setting a schema to force all tables and views to be created in this schema when clicking CTAS or CVAS in SQL Lab doesn't seem to be configurable by code or am I'm missing something?

![image](https://user-images.githubusercontent.com/43774417/179705161-e878ff0b-3d02-4661-a055-4469cec22b7a.png)

When I run `superset export_datasources` and search for "testing" I don't find anything.

**Comment by michadavid** [comment]
This issue should receive more attention.
